metpy/plots/skewt.py

```diff
@@ -791,20 +791,17 @@ def wind_vectors(self, u, v, **kwargs):
         return self.ax.quiver(center_position, center_position,
                               u, v, **quiver_args)
 
-    def plot_colormapped(self, u, v, c, bounds=None, colors=None, **kwargs):
+    def plot_colormapped(self, u, v, c, cint=None, colors=None, **kwargs):
```

Comment:
1. `cint` feels like a misnomer since we're not contouring. Maybe `intervals`?
2. This is technically an API break, and it's possible people are using `bounds` currently. Since we're already taking `**kwargs` we could do some processing within the method to look for both `bounds` and the new name, issuing a deprecation warning if `bounds` is passed?

Reply:
Fine with me on both issues. I'll update this.

Reply:
Followed the deprecation syntax for a kwarg used in #869. Let me know if I missed something though.